### DISCUSSION TITLE
Running Release note against a certain build/release?

### DISCUSSION THREAD

**Discussion by anaim3** [rfennell/AzurePipelines#1134] [Q&A] [answered]
Hi Richard,

Thank you so much for that tool, I have used it this week to generate our release notes, its very helpful.

I have a problem using the testing tool, its running fine but it doesn't run against the build or release I want..

I have a branch 34 ( build number 28307 ) and I want to compare it with branch 32 ( build 28305 )
so whatever number I put in the build-settings.json file it doesn't find any prs ( using the testing tool only )
```

    "TeamFoundationCollectionUri": "https://dev.azure.com/obzervr",
    "TeamProject": "Obzervr",
    "buildId": "28305", // branch 32
    "releaseDefinitionId": "",
    "releaseId": "",
    "environmentName":"Windows",
    "templatefile": "build-handlebars-template.md",
    "outputfile": "output.md",
```

Is there something I am doing wrong here? 

Same problem with the releases updating release-settings.json doesn't find any prs, I really tried all numbers, nothing change.

```

    "TeamFoundationCollectionUri": "https://dev.azure.com/obzervr",
    "TeamProject": "Obzervr",
    "buildId": 28284,
    "releaseDefinitionId": 180,
    "releaseId": 11857,
    "environmentName":"Windows",
    "templatefile": "release-handlebars-template.md",
    "outputfile": "output.md",
```

**Comment by rfennell** [comment, marked as answer]
I think you need a config with the `overrideBuildReleaseId` to define the oldest build

Something like

```
    "TeamFoundationCollectionUri": "https://dev.azure.com/obzervr",
    "TeamProject": "Obzervr",
    "buildId": "28307", 
    "overrideBuildReleaseId": "28305", // branch 32 - this is the compare to build/release
    "releaseDefinitionId": "",
    "releaseId": "",
    "environmentName":"Windows",
    "templatefile": "build-handlebars-template.md",
    "outputfile": "output.md",
```

**Comment by anaim3** [comment]
Hi Richard, that definitely helped..

I managed to run the tool against the release.. 
I failed to do that with the build but I will use the release anyways.

Thanks again.
